### DISCUSSION TITLE
Added new custom colors to $theme-colors map but related classes are not created for text and background colors

### DISCUSSION THREAD

**Discussion by rahuldev39** [twbs/bootstrap#36698] [General] [answered]
I added custom colors to the $theme-colors map, btn-, link- colors are generated perfectly but text- and bg- colors are not generated. 

### Describe the issue
my main.scss file is as follows.

@import "../node_modules/bootstrap/scss/functions";
@import "../node_modules/bootstrap/scss/variables";
@import "../node_modules/bootstrap/scss/maps";
@import "../node_modules/bootstrap/scss/mixins";
@import "../node_modules/bootstrap/scss/utilities";
@import "../node_modules/bootstrap/scss/utilities/api";
//custom-API file
@import './abstract/api';

and code in  _api.scss is as follows.

//custom-colors
$custom-theme-colors: (
  "altdark":  #314259,
  'altsecondary' : #F579A0
);
$theme-colors : map-merge($theme-colors, $custom-theme-colors);

### What operating system(s) are you seeing the problem on?
windows

### What version of Bootstrap are you using?
5.2.0-beta1

**Comment by florianlacreuse** [comment, marked as answer]
As explained in the docs:

* https://getbootstrap.com/docs/5.2/customize/sass/#importing
* https://getbootstrap.com/docs/5.2/customize/sass/#variable-defaults

You need to include utilities API last to generate classes based on the updated Sass map.

**Reply by rahuldev39** [reply]
It Works Thanks 👍
